CuitService/Controllers/TaxInfoController.cs

```diff
@@ -0,0 +1,50 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace CuitService.Controllers
+{
+    public class TaxInfoController
+    {
+        private readonly ILogger<TaxInfoController> _logger;
+
+        public TaxInfoController(ILogger<TaxInfoController> logger)
+        {
+            _logger = logger;
+        }
+
+        // TODO: validate CUIT before, in filter, binder, etc. And avoid
+        // primitive obsession in cuit parameter.
+        [HttpGet("/taxinfo/by-cuit/{cuit}")]
+        public async Task<TaxInfo> GetTaxInfoByCuit(string cuit)
+        {
+            return new TaxInfo()
+            {
+                ActividadPrincipal = "620100-SERVICIOS DE CONSULTORES EN INFORMÁTICA Y SUMINISTROS DE PROGRAMAS DE INFORMÁTICA",
+                Apellido = null,
+                CUIT = cuit,
+                CatIVA = "RI",
+                CatImpGanancias = "RI",
+                DomicilioCodigoPostal = "7600",
+                DomicilioDatoAdicional = null,
+                DomicilioDireccion = "CALLE FALSA 123 Piso:2",
+                DomicilioLocalidad = "MAR DEL PLATA SUR",
+                DomicilioPais = "AR",
+                DomicilioProvincia = "01",
+                DomicilioTipo = "FISCAL",
+                Error = false,
+                EstadoCUIT = "ACTIVO",
+                Message = null,
+                Monotributo = false,
+                MonotributoActividad = null,
+                MonotributoCategoria = null,
+                Nombre = null,
+                PadronData = null,
+                ParticipacionesAccionarias = true,
+                PersonaFisica = false,
+                RazonSocial = "RZS C.S. SA",
+                StatCode = 0
+            };
```

Comment:
@VicThorX, @javierburger, @cbernat, @fgchaio, @swidzinskiMS, what do you think about making honor to original service structure? or should we improve it?

Reply:
I think it's ok to maintain it.  I'm not sure about all the possible scenarios or how can those vary the object values. I think it would be easier to handle the original structure.

Reply:
Mmm I like strongly-typed POCO classes, but in this case, maybe its better to return the response of the external API, for example with:
![image](https://user-images.githubusercontent.com/9144465/86925385-349f2c00-c107-11ea-8e62-c7f54499d0e6.png)

Reply:
Mmm, I would prefer a mix, a robust system. A robust system is a system that is permissive with the inputs and strict with the outputs.

If we make it only a dynamic bridge and there is a breaking change in the original service, it will not be broken, but our clients will.

So, I prefer strongly types for output.